src/main/java/seedu/duke/Ui.java

```diff
@@ -0,0 +1,79 @@
+package main.java.seedu.duke;
+
+import java.util.Scanner;
+
+/**
+ * Provides user interaction for Duke.
+ */
+public class Ui {
+    private int lengthOfLine = 55;
+    private String line;
+    private Scanner sc;
+
+    public Ui() {
+        this.line = getHorizontalLine();
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * Outputs a horizontal line.
+     * @return An horizontal line in a string.
+     */
+    public String getHorizontalLine() {
+        String line = "";
+        for (int i = 0; i < lengthOfLine; i++) {
+            line = line + "-";
+        }
+        return line;
+    }
```

Comment:
I think the horizontal line should be a static variable and should only be created once